<launch>
    <arg name="veh" default="dori" doc="Name of robot_name. ex: megaman"/>
    <arg name="show_marker" default="true" doc="Specify if you want to see the marker"/>
    <arg name="show_image" default="true" doc="Specify if you want to see the image"/>
    <arg name="bb_len" default="500" doc="Specify length of bounding box"/>
    <arg name="bb_wid" default="250" doc="Specify width of bounding box"/>
    <arg name="show_bb" default="true" doc="Specify if bounding box should be shown"/>

    <param name="~veh" value="$(arg veh)"/>
    <param name="~show_marker" value="$(arg show_marker)" type="bool"/>
    <param name="~show_image" value="$(arg show_image)" type="bool"/>
    <param name="~bb_len" value="$(arg bb_len)" type="double"/>
    <param name="~bb_wid" value="$(arg bb_wid)" type="double"/>
    <param name="~show_bb" value="$(arg show_bb)" type="bool"/>


    <node name="obstacle_detection_node_visual" pkg="obst_avoid" type="obstacle_detection_node_visual.py" output="screen">
    </node>


</launch>